joint_state_controller:
  type: joint_state_controller/JointStateController

left_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: left_wheel_hinge

right_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: right_wheel_hinge

front_yaw_position_controller:
  type: effort_controllers/JointPositionController
  joint: front_yaw_joint
